modules/ROOT/pages/asciidoc-vs-markdown.adoc

```diff
@@ -1,10 +1,10 @@
-= AsciiDoc Versus Markdown
+= Compare AsciiDoc and Markdown
```

Comment:
@graphitefriction It's not a :boxing_glove: anymore? I'm joking, nice editing :ok_hand: